### PR TITLE
Fixes "Export Page to ..." errors

### LINKED ISSUE #71: Export Page to PDF / PNG no longer works
Love this function and can't find it in any other plugin! 

Appears it has stopped working since the update. 

Console is as follows:

11/13/16 9:30:04.247 AM Whole Page to Desktop as PDF (Sketch Plugin)[66227]: TypeError: doc.currentView().totalRectForLayers is not a function. (In 'doc.currentView().totalRectForLayers(all)', 'doc.currentView().totalRectForLayers' is undefined)
line: 4
sourceURL: /Users/adamxx/Library/Application Support/com.bohemiancoding.sketch3/Plugins/Sketch Commands.sketchplugin/Contents/Sketch/Export/Whole Page to Desktop as PDF.cocoascript
column: 50

### PR DESCRIPTION
**Pull request by bre7**
Closes #71

Fixes totalRectForLayers / requestWithRect_scale errors

ToDo: Refactor code to Utils

### CODE REVIEW

File: Sketch Commands.sketchplugin/Contents/Sketch/Export/Whole Page to Desktop as PDF.cocoascript [hunk 1]
```diff
@@ -1,8 +1,15 @@
 var onRun = function(context) {
   var doc = context.document
   var all = doc.currentPage().children()
```
Comment (bre7) [inline Sketch Commands.sketchplugin/Contents/Sketch/Export/Whole Page to Desktop as PDF.cocoascript]: Can we use `artboards()` instead of `children()` ?

**Comment by bre7** [comment]
Can we use `artboards()` instead of `children()` ?

**Comment by bomberstudios** [comment]
> Can we use `artboards()` instead of `children()` ?

You're assuming that

1. the page will always contain artboards
2. the page will contain nothing but artboards

both of which are certainly not true all of the time : )

Thanks for the PR, will merge as soon as I finish reviewing it!

**Comment by bre7** [comment]
👌 true.

I’ll push an update to the manifest.json to support auto-update (I bet it can be automated, if needed)

**Comment by bomberstudios** [comment]
Awesome, thanks!

**Pull request merged**